src/explore-education-statistics-admin/src/pages/publication/components/PublicationPublishedReleases.tsx

```diff
@@ -0,0 +1,171 @@
+  }, [releases, currentPage]);
+
+  const showMoreNumber =
+    pageSize < releases.length - currentReleases.length
+      ? pageSize
+      : releases.length - currentReleases.length;
+
+  useEffect(() => {
+    if (focusIndex) {
+      focusRef.current?.focus();
+    }
+  }, [focusIndex]);
+
+  return (
+    <>
+      <table
+        className="dfe-hide-empty-cells"
+        data-testid="publication-published-releases"
+      >
+        <caption
+          aria-live="polite"
+          aria-atomic
+          className="govuk-table__caption--m"
+        >
+          {`Published releases (${currentReleases.length} of ${releases.length})`}
+        </caption>
+        <thead>
+          <tr>
+            <th className="govuk-!-width-one-third">Release period</th>
+            <th className={styles.statusColumn}>
+              State{' '}
+              <ButtonText onClick={togglePublishedStatusGuidance.on}>
+                <InfoIcon description="Guidance on states" />
+              </ButtonText>
+            </th>
+            <th>Published date</th>
+            <th colSpan={2}>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {currentReleases.map((release, index) => (
+            <tr
+              className={styles.row}
+              key={release.id}
+              ref={focusIndex === index + 1 ? focusRef : undefined}
+              tabIndex={focusIndex === index + 1 ? -1 : undefined}
+            >
+              <td>{release.title}</td>
+              <td>
+                <Tag colour="green">Published</Tag>
+              </td>
+              <td>
+                {release.published && (
+                  <FormattedDate>{release.published}</FormattedDate>
+                )}
+              </td>
+              <td>
+                {release.permissions.canMakeAmendmentOfRelease && (
+                  <ButtonText onClick={() => setAmendReleaseId(release.id)}>
+                    Amend<VisuallyHidden> {release.title}</VisuallyHidden>
+                  </ButtonText>
+                )}
+              </td>
+              <td>
+                <Link
+                  to={generatePath<ReleaseRouteParams>(
+                    releaseSummaryRoute.path,
+                    {
+                      publicationId: release.publicationId,
+                      releaseId: release.id,
+                    },
+                  )}
+                >
+                  View<VisuallyHidden> {release.title}</VisuallyHidden>
+                </Link>
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+      {currentReleases.length < releases.length && (
+        <ButtonText
+          onClick={() => {
+            setFocusIndex(pageSize * currentPage + 1);
+            setCurrentPage(currentPage + 1);
+          }}
+        >
+          {`Show next ${showMoreNumber} published release${
+            showMoreNumber > 1 && 's'
+          }`}
```

Comment:
- If this is false we get `Show next x published releasefalse`
- If there is only one release, the sentence doesn't read well. Would suggest changing this to 'Show x more published releases' (handling the singular case of course)